macros/new_datacard_tt_ot.py

```diff
@@ -206,15 +504,23 @@ def write_plots(plotter, variable, output_dir):
             output_dir
             )
         )
-
+    
```

Comment:
OK for the following lines, yet if this does not change then one can loop only on `cfg.datacards_variables` for the systematics, see [this line](https://github.com/lucastorterotot/htt_plot/pull/1/files#diff-1fb34ac54bc23f9f0e8b9da7bbdf8d7eR213).